benchmark_medprompt.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+from benchmark import test
+from medprompt import MedpromptModule
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
```

Comment:
none of these getting used?

Reply:
Instead of typing the parameters in the script itself you can run the script and  specify the parameters using the cli command also, for that reason this part is used. I chose to avoid that part in the video. If you want I can add the clip where I run the script using this method.

Reply:
this script sample doesnt actually use the values passed to the cli currently

Reply:
Yes for the demo I entered the values that way, I will change that part.

Reply:
> Yes for the demo I entered the values that way, I will change that part.